spring-cloud-data-rest-resource/src/main/java/org/springframework/cloud/data/rest/resource/TaskDefinitionResource.java

```diff
@@ -30,6 +31,12 @@
 
 	private String dslText;
 
+	/**
+	 * Stream status (i.e. running, complete, etc).
+	 */
+	private String status;
+	
+
 	public TaskDefinitionResource() {
```

Comment:
is this no-arg constructor ever used? (there is no setter for name or status, so I don't see how a valid object would be created that way)